apps/staking/src/components/Home/index.tsx

```diff
@@ -16,6 +16,10 @@ import { Error as ErrorPage } from "../Error";
 import { Loading } from "../Loading";
 import { NoWalletHome } from "../NoWalletHome";
 
+const ONE_SECOND_IN_MS = 1000;
```

Comment:
we can have these in a global constants file to avoid repetition.

Reply:
Yeah, I'll refactor them in eventually, but it will go away when I clean up use-api anyways